src/screens/Home/QuickStart/QuickStartUIUtils.res

```diff
@@ -221,23 +221,50 @@ module SelectConnectorGrid = {
       Processors(ADYEN),
       Processors(CHECKOUT),
     ]->Array.filter(connector => {
-      !(typedConnectedConnectorList->Array.includes(connector))
+      let filteredValue = typedConnectedConnectorList->Array.find(item => {
+        QuickStartUtils.filterConnectorFromArray(item, connector)
+      })
+
+      switch filteredValue {
+      | Some(_value) => false
+      | None => true
+      }
+    })
+
+    let remainingConnectorList = connectorList->Array.filter(value => {
+      let popularConnectorListValue = popularConnectorList->Array.find(item => {
+        QuickStartUtils.filterConnectorFromArray(item, value)
+      })
+
+      let typedConnectedConnectorListValue = typedConnectedConnectorList->Array.find(item => {
+        QuickStartUtils.filterConnectorFromArray(item, value)
+      })
```

Comment:
If single line is there then {} is not needed

Reply:
Not a single line function